CoreArrangementPlugins/release/modules/ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/layer-by-time.md

```diff
@@ -20,5 +20,21 @@ the data in the same way, but puts transactions on a separate layer
 (between their source and destination nodes). Experiment with both
 options to see which one works best for you.
 
-<img src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/layer_by_time_parameters_gui_screenshot.jpg" alt="Layer by Time
-Interface" />
+<div style="text-align: center">
+    <img height=600 src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/layer_by_time_parameters_gui_screenshot.png" alt="Layer by Time Interface" />
+</div>
+<br />
+<br />
+
+Example:
+
+<div style="text-align: center">
+    <figure style = "display: inline-block">
+        <img height=400 src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/BeforeLayer.png" alt="Before" />
+        <figcaption>Before</figcaption>
+    </figure>
+    <figure style = "display: inline-block">
+        <img height=400 src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/AfterLayer.png" alt="After" />
+        <figcaption>After</figcaption>
```

Comment:
maybe a smaller graph might be better to use here since its hard to see much in the after screenshot